setup.cfg

```diff
@@ -1,4 +1,4 @@
 [pytest]
 python_files = *.py
-addopts = --verbose --ignore setup.py --ignore utide/robustfit.py --doctest-modules --cov-report term-missing --cov .
```

Comment:
@efiring My guess is that `--cov-report` and `--cov` were not recognized, right?  If so, that is probably because they depend on the [pytest-cov](https://pypi.python.org/pypi/pytest-cov) module.  That module provides code coverage reports.  It is not too important so it is OK to remove.